.github/workflows/test.yml

```diff
@@ -13,16 +13,17 @@ jobs:
 
     strategy:
       matrix:
-        ruby: [ '2.5', '2.6', '2.7.1' ]
+        ruby: [ '2.5', '2.6', '2.7', '3.0', '3.1' ]
```

Comment:
We should be able to drop a few Ruby versions. 

From this [doc](https://endoflife.software/programming-languages/server-side-scripting/ruby), Ruby 2.5 already reached EOL last year and 2.6 will reach it in two months. So supporting from 2.7 and above should be enough.